aiidalab_widgets_base/__init__.py

```diff
@@ -10,7 +10,8 @@
 from .databases import CodQueryWidget  # noqa
 #from .editors import editor  # noqa
 from .export import ExportButtonWidget  # noqa
-from .process import ProcessFollowerWidget, ProgressBarWidget, RunningCalcJobOutputWidget, SubmitButtonWidget  # noqa
+from .process import ProgressBarWidget, ProcessFollowerWidget, ProcessInputsWidget, ProcessOutputsWidget  # noqa
+from .process import ProcessCallStackWidget, RunningCalcJobOutputWidget, SubmitButtonWidget, ProcessReportWidget  # noqa
```

Comment:
If we add `# noqa` to basically every import then we can as well stop checking the style. What is the issue that is suppressed here?

Reply:
The issue was `wrong-import-position`. Made PR #92 to remove all the `# noqa`s, please have a look.

Reply:
Got it, #92 is approved.